docs/src/pages/index.tsx

```diff
@@ -25,7 +24,6 @@ export default function Home(): JSX.Element {
         <HomepageHeader />
         <div>
           <Welcome />
-          <Code />
```

Comment:
Removed this code section because it was being duplicated in three places. Now it is only in README and intro of the doc site.